### ISSUE TITLE
Does this work with v2.3 WAB?

### ISSUE THREAD

**Issue by ppenn** [cmndrbensisko/LocalLayer#60]
If I install LocalLayer by copying to widgets folder and modifying config.json, I can see the widget just fine in my map design. Clicking configure on the LocalLayer widget brings up that dialog just fine. But I can't save or cancel out of that. OK, Cancel and the X button in the top right of the dialog do nothing for me. Tried on Chrome & Edge, same result. No errors being spat out of the node window in the background, either. 

From Chrome Debugger on clicking OK, map.updatedLayerInfos is undefined :
            LayerInfos.getInstanceSync()._tables = this.map.updatedLayerInfos._tables;  

Uncaught TypeError: Cannot read property '_tables' of undefined
    at Object.<anonymous> (Setting.js?wab_dv=2.3:89)
    at Object.c [as destroy] (init.js:118)
    at Object.onClose (main.js?wab_dv=2.3:3796)
    at Object.onOk (main.js?wab_dv=2.3:3795)
    at Object.onClick (init.js:63)
    at Object.<anonymous> (Popup.js?wab_dv=2.3:443)
    at HTMLDivElement.<anonymous> (init.js:63)
(anonymous) @ Setting.js?wab_dv=2.3:89
c @ init.js:118
onClose @ main.js?wab_dv=2.3:3796
onOk @ main.js?wab_dv=2.3:3795
(anonymous) @ init.js:63
(anonymous) @ Popup.js?wab_dv=2.3:443
(anonymous) @ init.js:63

**Comment by RG333** [comment]
I'm having the same issue, was there any resolution to this?

**Comment by cmndrbensisko** [comment]
An odd question, but are you setting the widget to have "visibility: false" by default?

**Comment by cameronwallace** [comment]
I have... no go.
Any chance there's an update for 2.4 in progress?

**Comment by RG333** [comment]
Yes, I've set "visibility: false" and it seems like Barnaby Rockwell has had success. Wonder what I'm doing wrong...?
https://geonet.esri.com/thread/192424-locallayer-widget-and-wab-23?et=watches.email.outcome

**Comment by dh81** [comment]
Hi 
is there any solution for this issue, besides exporting and importing from an old WAB version 2.0 as in https://geonet.esri.com/thread/192424-locallayer-widget-and-wab-23?et=watches.email.outcome
I tried this LocalLayers version in WAB 2.4 !

**Comment by GBH09** [comment]
We are going to use Dynamic Layers on our own ArcGIS Server and Feature Layers , and we are going to use WAB 2.8, is Latest local Layer compatible with WAB 2.8? If not what needs to be done to make it compatible, Thanks
